upstream-community-operators/syndesis/1.11.0/manifests/syndesisoperator.1.11.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,844 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: syndesis-operator.1.11.0
+  namespace: placeholder
+  annotations:
+    capabilities: Seamless Upgrades
+    categories: Integration & Delivery
+    certified: "false"
+    createdAt: '2021-03-06T12:00:00Z'
+    containerImage: quay.io/syndesis/syndesis-operator:1.11.0
+    support: Syndesis
+    description: Manages the installation of Syndesis, a flexible and customizable
+      open source platform that provides core integration capabilities as a service.
+    repository: https://github.com/syndesisio/syndesis/
+    alm-examples: |
+      [{
+        "apiVersion": "syndesis.io/v1beta2",
+        "kind": "Syndesis",
+        "metadata": {
+          "name": "app"
+        },
+        "spec" : {
+          "addons": {
+            "jaeger": {
+              "enabled": true
+            },
+            "knative": {
+              "enabled": false
+            },
+            "ops": {
+              "enabled": false
+            },
+            "publicApi": {
+              "enabled": false
+            },
+            "todo": {
+              "enabled": false
+            }
+          }
+        }
+      }]
+spec:
+  displayName: Syndesis
+  description: |
+    ### Syndesis operator
+    Syndesis is a flexible and customizable, open source platform that provides core integration capabilities as a service.
+
+    This operator installs as well as configures the following syndesis components:
+    - syndesis-server
+    - syndesis-meta
+    - syndesis-ui
+    - syndesis-db
+    - syndesis-prometheus
+    - syndesis-proxy
+    - syndesis-oauthproxy
+
+    ### How to install
+    When the operator is installed (you have created a subscription and the operator is running in the selected namespace) create a new CR of Kind Syndesis (click the Create New button). The CR spec contains all defaults (see below).
+
+    ### CR Defaults
+    The CR definition is pretty simple and an empy CR will trigger a base installation.
+
+    Within the addons section, users are able to enable specific addons. The available addons at the moment are:
+    - jaeger: enable jaeger
+    - ops: enables monitoring, requires extra CRDs
+    - todo: a simple todo application
+
+    To enable addons, set "addon_name": {"enabled": true} in the CR.
+  keywords:
+  - camel
+  - integration
+  - syndesis
+  - fuse
+  - online
+  version: "1.11.0"
+  maturity: stable
+  maintainers:
+  - name: Syndesis team
+    email: syndesis@googlegroups.com
+  provider:
+    name: Syndesis team
+  replaces: syndesis-operator.1.9.0
```

Comment:
There is no version `1.9.0`. So please remove this line. You can start doing replaces when you release next version.

Reply:
Thanks @mvalarh, I've changed it to `syndesisoperator.1.9.0` which is here:

https://github.com/operator-framework/community-operators/blob/f9afba59feefb0fe099b8830b160df3bcfffd32d/community-operators/syndesis/1.9.0/syndesisoperator.1.9.0.clusterserviceversion.yaml#L4

Would that work?